File: Modul-5/Soal 2/PRAK502-2410817120019-NadyaRamadhani.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
    int angka = nilai1 - nilai2;
    if(angka<0){
        return -angka;
    }else{
        return angka;
    }
}

int mutlak(int angka){
    if(angka<0){
        return -angka;
    }
    return angka;
}

int main(){ 
    int a,b,c,d;
    scanf("%d",&a);
    scanf("%d",&b);
    scanf("%d",&c);
    scanf("%d",&d);
    int hasil = hitung(a,c) + hitung(b,d);
    printf("%d",mutlak(hasil));
    
    return 0;
}

File: Modul-5/Soal 2/PRAK502-2410817120019-NadyaRamadhani.py
def hitung(nilai1, nilai2):
    angka = nilai1 - nilai2
    if angka < 0:
        return -angka
    else:
        return angka

def mutlak(angka):
    if angka < 0:
        return -angka
    return angka

a, b, c, d = map(int, input(). split())
hasil = hitung(a, c) + hitung(b, d)
print(mutlak(hasil))